### PR TITLE
fix(j-s): fix ready for court mail sent to prosector

### PR DESCRIPTION
**Pull request by Arnarkari93**
[Asana](https://app.asana.com/0/1199153462262248/1202148289367416/f)

## What

- Fix body and subject for email sent out to prosecutors when a case is ready for court

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/src/app/formatters/formatters.ts [hunk 1]
```diff
@@ -134,6 +134,28 @@ export function formatCourtResubmittedToCourtSmsNotification(
   })
 }
 
+export function formatReadyForCourtEmailNotificationToProsecutor(
```
Comment (gudjong) [inline apps/judicial-system/backend/src/app/formatters/formatters.ts:137]: Consider keeping the naming convention. The name `formatProsecutorReadyForCourtEmailNotification` would be appropriate if you do.

File: apps/judicial-system/backend/src/app/modules/notification/notification.service.ts [hunk 2]
```diff
@@ -52,6 +52,7 @@ import { CaseEvent, EventService } from '../event'
 import { SendNotificationDto } from './dto/sendNotification.dto'
 import { Notification } from './models/notification.model'
 import { SendNotificationResponse } from './models/sendNotification.resopnse'
+import { formatReadyForCourtEmailNotificationToProsecutor } from '../../formatters/formatters'
```
Comment (gudjong) [inline apps/judicial-system/backend/src/app/modules/notification/notification.service.ts:55]: Consider adding the new export to the formatters barrel file and import it from there.

**Review by gudjong — CHANGES_REQUESTED** [review]

File: apps/judicial-system/backend/src/app/modules/notification/notification.service.ts (hunk 2 above)
Comment (Arnarkari93) [reply apps/judicial-system/backend/src/app/modules/notification/notification.service.ts]: Adressed 0f841917846b9d78c85b4332b274739406656f7a

File: apps/judicial-system/backend/src/app/formatters/formatters.ts (hunk 1 above)
Comment (Arnarkari93) [reply apps/judicial-system/backend/src/app/formatters/formatters.ts]: Adressed 0f841917846b9d78c85b4332b274739406656f7a

**Review by gudjong — APPROVED** [review]

**Pull request merged**